brave-jersey2/src/test/java/com/github/kristofa/brave/jersey2/ReporterForTesting.java

```diff
@@ -0,0 +1,38 @@
+package com.github.kristofa.brave.jersey2;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+import zipkin.Span;
+import zipkin.reporter.Reporter;
+
+public class ReporterForTesting implements Reporter<Span> {
```

Comment:
personal taste.. I prefer using a lambda reporter over a collection present in the actual test class used.